### PR TITLE
print compile output

### PR DESCRIPTION
**Pull request by makk94**
re: https://detectors.atlassian.net/browse/DET-41

needs: https://github.com/preludeorg/service/pull/688

Example output (personal stack):
```
mahinakaholokula@Mahinas-MacBook-Pro my-prelude % prelude --profile personal build upload b0f2c037-40d7-4a73-98bb-7025978904e8.go
Uploading b0f2c037-40d7-4a73-98bb-7025978904e8.go
Compiling b0f2c037-40d7-4a73-98bb-7025978904e8.go
[
  {
    "name": "b0f2c037-40d7-4a73-98bb-7025978904e8_linux-x86_64",
    "steps": [
      {
        "step": "COMPILE",
        "status": 0,
        "output": "",
        "duration": "0.722"
      },
      {
        "step": "SCAN",
        "status": 0,
        "output": [],
        "duration": "0.001"
      },
      {
        "step": "PUBLISH",
        "status": 0,
        "output": "",
        "duration": "0.124"
      }
    ]
  },
  {
    "name": "b0f2c037-40d7-4a73-98bb-7025978904e8_linux-arm64",
    "steps": [
      {
        "step": "COMPILE",
        "status": 0,
        "output": "",
        "duration": "0.577"
      },
      {
        "step": "SCAN",
        "status": 0,
        "output": [],
        "duration": "0.001"
      },
      {
        "step": "PUBLISH",
        "status": 0,
        "output": "",
        "duration": "0.165"
      }
    ]
  },
  {
    "name": "b0f2c037-40d7-4a73-98bb-7025978904e8_darwin-x86_64",
    "steps": [
      {
        "step": "COMPILE",
        "status": 0,
        "output": "",
        "duration": "0.711"
      },
      {
        "step": "SCAN",
        "status": 0,
        "output": [],
        "duration": "0.001"
      },
      {
        "step": "PUBLISH",
        "status": 0,
        "output": "",
        "duration": "0.126"
      }
    ]
  },
  {
    "name": "b0f2c037-40d7-4a73-98bb-7025978904e8_darwin-arm64",
    "steps": [
      {
        "step": "COMPILE",
        "status": 0,
        "output": "",
        "duration": "0.687"
      },
      {
        "step": "SCAN",
        "status": 0,
        "output": [],
        "duration": "0.001"
      },
      {
        "step": "PUBLISH",
        "status": 0,
        "output": "",
        "duration": "0.119"
      }
    ]
  },
  {
    "name": "b0f2c037-40d7-4a73-98bb-7025978904e8_windows-x86_64",
    "steps": [
      {
        "step": "COMPILE",
        "status": 0,
        "output": "",
        "duration": "0.718"
      },
      {
        "step": "SCAN",
        "status": 0,
        "output": [],
        "duration": "0.001"
      },
      {
        "step": "PUBLISH",
        "status": 0,
        "output": "",
        "duration": "0.139"
      }
    ]
  }
]

```

### CODE REVIEW

File: python/cli/prelude_cli/views/build.py [hunk 1]
```diff
@@ -97,8 +97,10 @@ def test_id():
 
     def upload(p: Path):
         with open(p, 'r') as source_code:
-            controller.upload(test_id=identifier, filename=p.name, code=source_code.read())
+            out = controller.upload(test_id=identifier, filename=p.name, code=source_code.read())
             click.secho(f'Uploaded {path}', fg='green')
+            click.secho(f'Compiled {path}', fg='green')
```
Comment (privateducky) [inline python/cli/prelude_cli/views/build.py:102]: let's kill the secho here -- I'm aiming to make the CLI only JSON, no other print outs

**Review by privateducky — CHANGES_REQUESTED** [review]
1 small thing

File: python/cli/prelude_cli/views/build.py (hunk 1 above)
Comment (makk94) [reply python/cli/prelude_cli/views/build.py]: done

**Review by privateducky — APPROVED** [review]

**Comment by packplusplus** [comment]
👀

**Comment by privateducky** [comment]
> 👀

This is dependent on a change set that is in review / undergoing changes in Service right now.

**Pull request closed without merging**